ExtraDry/Sample.Data/Extensions/TaxonomyExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using System;
+
+namespace Sample.Data {
+    public static class TaxonomyExtensions {
+        /// <summary>
+        /// Moves a subtree from one point in the heirarchy to another.
+        /// </summary>
+        public static async Task MoveSubtree(this ITaxonomyEntity subtreeRootToMove, ITaxonomyEntity newParent, DbContext context)
```

Comment:
<remarks>
This does not use entity tracking and does a bulk-update to the database.  It will participate in enclosing transactions and care should be taken that the Ancestors and Descendants are not otherwise changed in the same transaction.
</remarks>